### ISSUE TITLE
Accessibility: Color of text and icons in block header fails contrast test

### ISSUE THREAD

**Issue by glombek** [PerplexDigital/Perplex.ContentBlocks#15]
Colors ought to have a contrast ratio of 7:1 against their background for legibility. The pale grey used here is difficult to see.

![image](https://user-images.githubusercontent.com/3120611/83775130-82df8c00-a67e-11ea-9a0e-f1b56cb94dab.png)

**Comment by PerplexDaniel** [comment]
Closed due to 4 weeks of inactivity.
Feel free to reopen with new input.

**Comment by glombek** [comment]
This issue should not be closed due to inactivity - these will take time to fix but remain severe accessibility issues.

**Comment by PerplexDaniel** [comment]
I'm a bit wary of leaving issues open forever, then we can end up with so many open issues which never get resolved. 
The policy for this repo is to close any issues after ~1 month of inactivity. This does not mean it will never be resolved, I see it more as a way to "archive" them and keep things clean. Perhaps there is a better way to go about this but I want to try this first.

This can potentially lead to duplicated issues, but if people search first they should find this and can reopen it for you. 
Like I said though, feel free to reopen but I prefer that happens when a PR is attached so we can deal with it instead of just leaving it open in an idle state.
